build-targets/steelseries-rival310.txt

```diff
@@ -1,2 +1,4 @@
 [properties]
-linker = 'stm32f103x6'
+target = 'rival310'
+linker = 'rival310-oem-boot'
```

Comment:
I think we should have a different target for this case.

Reply:
i agree, but it was not the point of this pr, i just wanted to have it working or now.